spec/models/user_spec.rb

```diff
@@ -13,16 +13,19 @@
       .source(:organization)
   end
 
-  it { should validate_presence_of(:fb_id) }
-  it { should validate_presence_of(:first_name) }
-  it { should validate_presence_of(:last_name) }
-  it { should validate_uniqueness_of(:fb_id) }
-  it { should validate_uniqueness_of(:email) }
+  it "should validate that email is case-sensitively unique" do
+    user = users(:bob_user)
+    new_user = User.new(first_name: "John", last_name: "Doe", email: user.email)
+    expect(new_user.valid?).to be_falsey
+    new_user.update_attribute(:email, user.email.upcase)
+    expect(new_user.valid?).to be_falsey
+  end
```

Comment:
YESSSSS

Reply:
that being said, if you would like to, can you give it a go for adding tests for the other user methods you have added?

Reply:
fosheez. on it

Reply:
actually ill put this on another branch. We can use https://github.com/omniauth/omniauth/wiki/Integration-Testing